libraries/ESP8266WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -165,6 +165,51 @@ class StaticRequestHandler : public RequestHandler<ServerType> {
     size_t _baseUriLength;
 };
 
+template<typename ServerType>
+class StaticRequestETagHandler
+    :
+public StaticRequestHandler<ServerType> {
+
+    using SRH = StaticRequestHandler<ServerType>;
+
+    using SRH::SRH;
+
+    using WebServerType = ESP8266WebServerTemplate<ServerType>;
+
+    public:
+    bool handle(WebServerType& server, HTTPMethod requestMethod, const String & requestUri) override {
+        if (!SRH::canHandle(requestMethod, requestUri)){
+            return false;
+        }
+
+        if(server.header("If-None-Match") == SRH::_cache_header){
+            // Serial.println("Sending 304!!!");
```

Comment:
Can we please drop the commented debug messages, or bracket the prints with the `DBGWS` macro based off of `DEBUG_ESP_HTTP_SERVER`

Reply:
I see this is taken care of in latest PR, thanks!